tests/basic/errorlog_usec.phpt

```diff
@@ -0,0 +1,29 @@
+--TEST--
+Microseconds in error log file
+?>
+--INI--
+error_log=error_usec_test.log
+error_log_usec=On
+--FILE--
+<?php
+
+define('LOG_FILENAME', ini_get("error_log"));
+
+if (file_exists(LOG_FILENAME)) {
+    unlink(LOG_FILENAME);
+}
+
+error_log("hello world");
+
+assert(file_exists(LOG_FILENAME));
+
+printf("errorlog contents\n%s", file_get_contents(LOG_FILENAME));
```

Comment:
```suggestion
var_dump(file_exists(LOG_FILENAME));

error_log("hello world");

printf("errorlog contents\n%s", file_get_contents(LOG_FILENAME));
```

Reply:
@negram in the latest commit you still try to unlink the file, which is not wanted, the file should not exist

Reply:
@mvorisek, why do you feel like it shouldn't exist?

Reply:
I did an expetiment now and I can confirm it is not created by default no error is logged by default. So what creates it on your side?

If the file is there from this tests, the CLEAN section is responsible for deleting it.

Reply:
Actually, I just checked on the newly built PHP and the file is created if we define `error_log` option:
```bash
$ ./sapi/cli/php -derror_log=error_usec_test.log -r 'error_log("hello world"); var_dump(file_exists("error_usec_test.log"));'
bool(true)

$ cat error_usec_test.log
[25-Nov-2022 14:44:53 UTC] hello world
```

And after-commit test runs verified that `file_exists()` returned `true` if I understand everything correctly:
[link to test-file](https://github.com/php/php-src/pull/9844/files#diff-03f5efa1e91b7c457f332e6586749bb4de28a346a071310d6d08443be236ffe0R27) 
[link to the test-run](https://github.com/php/php-src/actions/runs/3548456725/jobs/5959682787) 

Maybe I understand something wrongly? Could it be the case that on your side, there some custom `error_reporting` settings take an effect here?

Reply:
you can remove `var_dump(file_exists(LOG_FILENAME));` as long as the file is created, but empty

Reply:
Agree, it doesn't actually check anything. If the file is not created, we'll have a warning from `file_get_contents()`